File: c/1099.c
#include <stdio.h>

int main(){

    int n;
    scanf("%d", &n);

    for (int i=0;i<n;i++){
        int x,y,cont = 0, aux;
        scanf("%d %d", &x, &y);
        if (y>x){
            aux = y;
            y=x;
            x=aux;
        }
        for (int i=y+1;i<x;i++){
            if (i%2 != 0){
                cont+=i;
            }
        }
        printf("%d\n", cont);
    }
}

File: js/1099.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')

lines = lines.map(function(item){
    return item.split(' ').map(x => parseInt(x))
})

n = lines[0][0]
for(i=1;i<=n;i++){
    x = lines[i][0]
    y = lines[i][1]
    
    cont=0
    if (y>x){
        aux = y
        y = x
        x = aux
    }
    for(j=y+1;j<x;j++){
        if (j%2!=0){
            cont+=j
        }
    }
    console.log(cont+'')
}